lib/rules/objectID.js

```diff
@@ -15,7 +15,7 @@ module.exports = function({ schema, messages, index }, path, context) {
 			return;
 		}
 
-		${schema.convert ? "return new ObjectID(value)" : "return value;" } ;
+		${schema.convert ? "return new ObjectID(value)" : "return value.toString();" } ;
```

Comment:
We can't modify the init-value when convert is false. add a property named `toString`. like:
```js
schema = {
    id: { type: "objectID", toString: true }
}
```
Also add a test

Reply:
I though this case
let me explains:
ObjectID has 2 state: ObjectID or HexString
If we need to use ObjectID, I must sure that `convert: true` because in the case we pass a param in String or ObjectID instance, the validate code above `if (!ObjectID.isValid(value))` just make sure that param should match `/^[0-9a-f]{24}$/`. Means, if you need it exactly Instance of ObjectID or String, you must force `ObjectID(param)` or `param.toString()` before using it.
But in real, I have never met case `ObjectID(param)`, so I think it's not popular case. So convert toString should be set as default

Reply:
I personally use `ObjectID(param)`.  It's essential for those who use MongoDB native driver directly (like me).
We have also stated in the documentation that the FV doesn't modify the init-value by default. This is contrary to what the user expects.

Reply:
Also this change is breaking, We must avoid breaking changes as much as possible

Reply:
got it
but toString is a default object method, may I change to 
```js
schema = {
    id: { type: "objectID", string: true }
}
```

Reply:
an if both of `convert: true` and `string: true`, we should ignore `string` or throw an error?

Reply:
schema = {
    id: { type: "objectID", convert: "hexString" }
}

Isn't it better?

Reply:
I know that but I think `schema.convert` will be true if you make a typo

Reply:
I prefer @erfanium idea as using `convert: true` and `convert: "hexString"`

Reply:
@0x0a0d
```js
if (schema.convert === true) src.push("return new ObjectID(value)")
else if (schema.convert === "hexString") src.push("return value.toString()")
else src.push("return value")
```
OK?